.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+### My suggestion
+
+[describe here]
+
+
+
+
+----
+
+<p align="center"><i>comments like +1 me too will be deleted</i><br><b>&#x25BC;Vote with reaction&#x25BC;</b></p>
```

Comment:
I don't think it's right to promise that useless comments will be deleted, because that commits some unspecified group of people to the task of moderating all comments and deleting the useless ones, without first creating any kind of system to ensure that will happen.

Instead I think it should:

1. tell users not to add those kinds of comments
2. *clearly* explains to them what to do instead ("Vote with reaction" is a good start, but I think it would be helpful to make it more explicit, since many users will not know what a "reaction" is)
3. link to a document which explains *why* they should follow this advice (perhaps #640 in the first instance, although I think it would be worth writing a better explanation)

Reply:
Not unless a github bot application is made to detect them on post and automatically delete said comments.

Reply:
It's true that a bot could take care of the deletion (and I'd personally be fine with that), but that would still be trying to put a bandaid on the symptoms rather than treating the cause.  Ideally we need both.

Reply:
Maybe @isaacs would be interested in installing https://github.com/apps/reaction/ to implement the conversion to reactions while @GitHub is _(presumably)_ solving this for real?